layers/vk_mem_alloc.h

```diff
@@ -16694,7 +16702,7 @@ VkResult vmaCreateImage(
         pImageCreateInfo->mipLevels == 0 ||
         pImageCreateInfo->arrayLayers == 0)
     {
-        return VK_ERROR_VALIDATION_FAILED_EXT;
+       return VK_ERROR_VALIDATION_FAILED_EXT;
```

Comment:
You've got some weird white space changes around this and the 3 instances of VK_ERROR_VALIDATION_FAILED_EXT above this.